### PR TITLE
Merge fluent-plugin-mysql-bulk.

### PR DESCRIPTION
**Pull request by toyama0919**
Hello.

[toyama0919/fluent-plugin-mysql-bulk](https://github.com/toyama0919/fluent-plugin-mysql-bulk) merge this repository.

This PR works mysql and mysql-bulk.

But warn log message「This plugin deprecated. You should use mysql_bulk」, if use mysql plugin. 

Thanks :blush:

### CODE REVIEW

File: lib/fluent/plugin/out_mysql.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def initialize
   def configure(conf)
     super
 
+    $log.warn "[mysql] This plugin deprecated. You should use mysql_bulk."
```
Comment (tagomoris) [inline lib/fluent/plugin/out_mysql.rb]: use `log.warn` to use plugin logger.

**Comment by toyama-h** [comment]
Thanks.

same fix applied to mysql-bulk.

**Comment by tagomoris** [comment]
:+1:
I added you to collaborators and gem owners. Please merge and release new version.

**Comment by toyama0919** [comment]
:blush:

**Comment by toyama0919** [comment]
released.

[fluent-plugin-mysql | RubyGems.org | your community gem host](https://rubygems.org/gems/fluent-plugin-mysql/)

**Comment by tagomoris** [comment]
:cherry_blossom:

**Pull request merged**